environments/dev/terraform/clinvar.tf

```diff
@@ -18,7 +18,7 @@ module clinvar {
   k8s_scaled_cluster_max_size = 3
   k8s_scaled_machine_type = "n1-standard-1"
   k8s_static_cluster_size = 2
-  reader_groups = ["clingendevs@broadinstitute.org"]
+  reader_groups = ["clingendevs@broadinstitute.org", "jade-k8-sa@broad-jade-dev.iam.gserviceaccount.com"]
```

Comment:
I remember now: This won't work as-is because Google's APIs force you to distinguish sharing with a user vs. a service account vs. a group. `reader_groups` assumes all the strings are groups.

I think the best way to handle this is to add a new `jade_repo_email` argument to the processing-project module, and then make a new IAM resource for it.